### ISSUE TITLE
No matching distribution found for mediapipe-silicon==0.8.11

### ISSUE THREAD

**Issue by chenjingyiyaya** [cansik/mediapipe-silicon#15]
When I run the command  
"/Applications/Blender.app/Contents/Resources/3.5/python/bin/python3.10 -m pip install mediapipe-silicon==0.8.11 --target /Applications/Blender.app/Contents/Resources/3.5/python/lib/python3.10/site-packages", the errors that reported is as follows:
ERROR: Could not find a version that satisfies the requirement mediapipe-silicon==0.8.11 (from versions: none)
ERROR: No matching distribution found for mediapipe-silicon==0.8.11.

Is it because of my system version？

My MAC: macOS Big Sur 11.3.1, MacBook Pro (13-inch, M1, 2020), Apple M1

**Comment by cansik** [comment]
No, it is because there is no build for python version 3.5, which your blender installation is using.

**Comment by cansik** [comment]
Could it be that blender is using rosetta2? Otherwise this should be possible, but I never tried to install it into blender. I am going to try it out today.

**Comment by chenjingyiyaya** [comment]
Although I don't know the exact reason, I upgraded my macOS to version 12, and then it worked when I retried. Perhaps it's because the system update also cleared all my Python environments.

**Comment by chenjingyiyaya** [comment]
thanks for your patience~
